docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+env_file:
```

Comment:
This tag must be in the service space. The variables in environment tag should be removed (they are duplicated)